feature/agenda/data/src/main/java/com/sergius/agenda/data/mappers/EventMapper.kt

```diff
@@ -0,0 +1,29 @@
+package com.sergius.agenda.data.mappers
+
+import com.sergius.agenda.data.dto.CreateEventRequest
+import com.sergius.agenda.data.dto.EventDto
+import com.sergius.core.domain.model.Event
+import java.time.Instant
+
+fun Event.toCreateEventRequest() = CreateEventRequest(
+    id = id,
+    title = title,
+    description = description,
+    from = Instant.ofEpochMilli(from).toString(),
+    to = Instant.ofEpochMilli(to).toString(),
+    remindAt = Instant.ofEpochMilli(remindAt).toString(),
+    updatedAt = Instant.ofEpochMilli(updatedAt ?: 0).toString(),
```

Comment:
I just noticed the `updatedAt ?: 0` you have there -  that basically means if it’s missing, you’ll end up sending 1970-01-01T00:00:00Z to the server 😅. That might not really reflect what you want. A cleaner way is to make it nullable and only send it when it’s actually set, something like:

`updatedAt = updatedAt?.let { Instant.ofEpochMilli(it).toString() }`